api/db/migrate/20210102222850_create_plan_changelogs.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePlanChangelogs < ActiveRecord::Migration[6.0]
+  def change
+    create_table :plan_changelogs do |t|
+      t.string :log, null: false
+      t.bigint :author_id, null: false
```

Comment:
author_id is needed so we can use this to determine if the current user is the author of the log or not without relying on the name (can be duplicate)